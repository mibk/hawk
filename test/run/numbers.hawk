BEGIN {
	print 0xFF, 0Xba
	print 2011.003
	print 888777
}
