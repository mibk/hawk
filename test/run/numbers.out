255 186
2011.003
888777
